--- plugin/gtk/Makefile.am.link	2011-09-22 13:53:15.396340012 +0800
+++ plugin/gtk/Makefile.am	2011-09-22 13:53:36.088210688 +0800
@@ -37,7 +37,7 @@
 
 libgvplugin_gtk_la_LDFLAGS = -version-info @GVPLUGIN_VERSION_INFO@
 libgvplugin_gtk_la_SOURCES = $(libgvplugin_gtk_C_la_SOURCES)
-libgvplugin_gtk_la_LIBADD = $(GTK_LIBS)
+libgvplugin_gtk_la_LIBADD = $(GTK_LIBS) -lX11
 
 if WITH_WIN32
 libgvplugin_gtk_la_LDFLAGS += -no-undefined
--- plugin/gtk/Makefile.in.link	2011-09-22 13:53:22.628204613 +0800
+++ plugin/gtk/Makefile.in	2011-09-22 13:54:21.718204654 +0800
@@ -510,7 +510,7 @@
 libgvplugin_gtk_la_LDFLAGS = -version-info @GVPLUGIN_VERSION_INFO@ \
 	$(am__append_1) $(am__append_2)
 libgvplugin_gtk_la_SOURCES = $(libgvplugin_gtk_C_la_SOURCES)
-libgvplugin_gtk_la_LIBADD = $(GTK_LIBS)
+libgvplugin_gtk_la_LIBADD = $(GTK_LIBS) -lX11
 EXTRA_DIST = gtk.glade gtk.gladep 
 all: all-am
 
--- cmd/dot/Makefile.am.link	2011-12-05 01:55:09.196840925 +0800
+++ cmd/dot/Makefile.am	2011-12-05 01:55:38.046841679 +0800
@@ -116,8 +116,8 @@
 endif
 
 if WITH_PANGOCAIRO
-dot_static_LDADD += $(top_builddir)/plugin/pango/.libs/libgvplugin_pango_C.a $(PANGOCAIRO_LIBS)
-dot_builtins_LDADD += $(top_builddir)/plugin/pango/libgvplugin_pango.la $(PANGOCAIRO_LIBS)
+dot_static_LDADD += $(top_builddir)/plugin/pango/.libs/libgvplugin_pango_C.a $(PANGOCAIRO_LIBS) $(PANGOFT2_LIBS)
+dot_builtins_LDADD += $(top_builddir)/plugin/pango/libgvplugin_pango.la $(PANGOCAIRO_LIBS) $(PANGOFT2_LIBS)
 endif
 
 EXTRA_DIST = Makefile.old dot.1 dot.1.pdf osage.1 osage.1.pdf patchwork.1 patchwork.1.pdf dot.vcproj
--- cmd/dot/Makefile.in.link	2011-05-07 11:27:39.000000000 +0800
+++ cmd/dot/Makefile.in	2011-12-05 02:07:36.536493154 +0800
@@ -53,8 +53,8 @@
 @WITH_LIBGD_TRUE@@WITH_MYLIBGD_TRUE@am__append_5 = $(top_builddir)/plugin/gd/libgvplugin_gd.la $(top_builddir)/lib/gd/libgvgd_C.la $(FONTCONFIG_LIBS) $(FREETYPE2_LIBS) $(JPEG_LIBS) $(PNG_LIBS) $(Z_LIBS)
 @WITH_LIBGD_TRUE@@WITH_MYLIBGD_FALSE@am__append_6 = $(top_builddir)/plugin/gd/.libs/libgvplugin_gd_C.a $(GD_LIBS) $(FONTCONFIG_LIBS) $(FREETYPE2_LIBS) $(JPEG_LIBS) $(PNG_LIBS) $(Z_LIBS)
 @WITH_LIBGD_TRUE@@WITH_MYLIBGD_FALSE@am__append_7 = $(top_builddir)/plugin/gd/libgvplugin_gd.la $(GD_LIBS) $(FONTCONFIG_LIBS) $(FREETYPE2_LIBS) $(JPEG_LIBS) $(PNG_LIBS) $(Z_LIBS)
-@WITH_PANGOCAIRO_TRUE@am__append_8 = $(top_builddir)/plugin/pango/.libs/libgvplugin_pango_C.a $(PANGOCAIRO_LIBS)
-@WITH_PANGOCAIRO_TRUE@am__append_9 = $(top_builddir)/plugin/pango/libgvplugin_pango.la $(PANGOCAIRO_LIBS)
+@WITH_PANGOCAIRO_TRUE@am__append_8 = $(top_builddir)/plugin/pango/.libs/libgvplugin_pango_C.a $(PANGOCAIRO_LIBS) $(PANGOFT2_LIBS)
+@WITH_PANGOCAIRO_TRUE@am__append_9 = $(top_builddir)/plugin/pango/libgvplugin_pango.la $(PANGOCAIRO_LIBS) $(PANGOFT2_LIBS)
 subdir = cmd/dot
 DIST_COMMON = $(srcdir)/Makefile.am $(srcdir)/Makefile.in
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
