--- graphviz-2.28.0/config/config_ruby.rb.ruby19~	2011-02-18 22:25:56.000000000 +0100
+++ graphviz-2.28.0/config/config_ruby.rb	2012-02-16 02:59:05.410021805 +0100
@@ -5,12 +5,12 @@ CONFIG = Config::MAKEFILE_CONFIG
 
 case ARGV[0]
 when "INCLUDES"
-    puts Config::expand(CONFIG["archdir"])
+    puts Config::expand(RbConfig::CONFIG["archdir"])
 when "lib"
-    puts Config::expand(CONFIG["libdir"])
+    puts Config::expand(RbConfig::CONFIG["libdir"])
 when "archsitelib"
-    puts Config::expand(CONFIG["sitearchdir"])
+    puts Config::expand(RbConfig::CONFIG["vendorarchdir"])
 when "sitelib"
-    puts Config::expand(CONFIG["sitedir"])
+    puts Config::expand(RbConfig::CONFIG["sitedir"])
 end
 
--- graphviz-2.28.0/configure.ac.ruby19~	2012-02-16 03:00:56.579170774 +0100
+++ graphviz-2.28.0/configure.ac	2012-02-16 03:00:59.489174708 +0100
@@ -1510,12 +1510,12 @@ else
       if test "x$RUBY" = "x"; then
 	use_ruby="No (ruby not available)"
       else
-        RUBY_INCLUDES="-I`$RUBY $srcdir/config/config_ruby.rb INCLUDES`"
+        RUBY_INCLUDES="$(pkg-config --cflags ruby-1.9)"
 	# hack for powerpc-darwin8 (10.4)
 	if test "x$DARWIN" = "xyes"; then
 	  RUBY_INCLUDES=`echo $RUBY_INCLUDES | sed 's/powerpc/universal/'`
 	fi
-        RUBY_LIBS="-L`$RUBY $srcdir/config/config_ruby.rb lib` `$RUBY -rrbconfig -e \"puts Config::CONFIG[['LIBRUBYARG_SHARED']]\"`"
+        RUBY_LIBS="$(pkg-config --libs ruby-1.9)"
         RUBY_INSTALL_DIR="`$RUBY $srcdir/config/config_ruby.rb archsitelib`"
         save_CPPFLAGS=$CPPFLAGS
         CPPFLAGS="$CPPFLAGS $RUBY_INCLUDES"
