--- graphviz-2.36.0/configure.ac.orig	2014-03-13 09:21:41.329592136 +0000
+++ graphviz-2.36.0/configure.ac	2014-03-13 09:22:11.929593115 +0000
@@ -1581,7 +1581,7 @@
 	    if test "x$DARWIN" = "xyes"; then
 	      RUBY_CFLAGS=`echo $RUBY_CFLAGS | sed 's/powerpc/universal/'`
 	    fi
-            RUBY_LIBS="-L`$RUBY $srcdir/config/config_ruby.rb lib` `$RUBY -rrbconfig -e \"puts Config::CONFIG[['LIBRUBYARG_SHARED']]\"`"
+	    RUBY_LIBS="$(pkg-config --libs ruby-1.9)"	
             RUBY_INSTALL_DIR="`$RUBY $srcdir/config/config_ruby.rb vendorarchdir`"
 	    if test "x$RUBY_INSTALL_DIR" = "x"; then
 	  	  RUBY_INSTALL_DIR="`$RUBY $srcdir/config/config_ruby.rb sitearchdir`"
