diff -up graphviz-stable_release_2.40.1/tclpkg/gv/gv_perl.cpp.omv~ graphviz-stable_release_2.40.1/tclpkg/gv/gv_perl.cpp
diff -up graphviz-stable_release_2.40.1/tclpkg/gv/Makefile.am.omv~ graphviz-stable_release_2.40.1/tclpkg/gv/Makefile.am
--- graphviz-stable_release_2.40.1/tclpkg/gv/Makefile.am.omv~	2018-05-14 20:23:07.532892603 +0200
+++ graphviz-stable_release_2.40.1/tclpkg/gv/Makefile.am	2018-05-14 20:23:24.613491180 +0200
@@ -11,7 +11,8 @@ AM_CPPFLAGS = \
 	-I$(top_srcdir)/lib/common \
 	-I$(top_srcdir)/lib/cgraph \
 	-I$(top_srcdir)/lib/cdt \
-	-I$(top_srcdir)/lib/pathplan
+	-I$(top_srcdir)/lib/pathplan \
+	-I/usr/lib64/perl5/CORE
 
 LIBS = -lc
 
