--- a/plugin/gtk/Makefile.am~	2013-02-14 21:27:39.000000000 +0800
+++ b/plugin/gtk/Makefile.am	2013-05-27 11:22:45.127287033 +0800
@@ -37,7 +37,7 @@
 
 libgvplugin_gtk_la_LDFLAGS = -version-info @GVPLUGIN_VERSION_INFO@
 libgvplugin_gtk_la_SOURCES = $(libgvplugin_gtk_C_la_SOURCES)
-libgvplugin_gtk_la_LIBADD = $(GTK_LIBS)
+libgvplugin_gtk_la_LIBADD = $(GTK_LIBS) -lX11
 
 if WITH_WIN32
 libgvplugin_gtk_la_LDFLAGS += -no-undefined
--- ./configure.ac.orig	2017-10-08 11:55:15.622878825 +0300
+++ ./configure.ac	2017-10-08 11:56:08.642875683 +0300
@@ -2775,7 +2775,7 @@ if test "x$use_gd" = "x"; then
           AC_MSG_WARN(Optional GD library not available)
           use_gd="No (library not found)"
 	  with_libgd="no"
-        ], $GD_LIBS)
+        ], $GDLIB_LIBS)
       fi
       LDFLAGS=$save_LDFLAGS
       CPPFLAGS=$save_CPPFLAGS
diff --git a/tclpkg/tcldot/Makefile.am b/tclpkg/tcldot/Makefile.am
index 58c88f4..e636cdc 100644
--- a/tclpkg/tcldot/Makefile.am
+++ b/tclpkg/tcldot/Makefile.am
@@ -96,7 +96,7 @@ endif
 
 if WITH_LIBGD
 libtcldot_builtin_la_LIBADD += $(top_builddir)/plugin/gd/libgvplugin_gd_C.la
-libtcldot_builtin_la_LIBADD += $(GD_LIBS)
+libtcldot_builtin_la_LIBADD += $(GDTCLFT_LIBS) $(GDLIB_LIBS)
 endif
 
 libtcldot_builtin_la_LIBADD += $(EXPAT_LIBS) $(SOCKET_LIBS) $(IPSEPCOLA_LIBS) $(MATH_LIBS)
